stock_picking_package_preparation_line/models/stock_picking_package_preparation_line.py

```diff
@@ -206,7 +206,8 @@ def action_put_in_pack(self):
             for line in package.line_ids:
                 # ----- If line has 'move_id' this means we don't need to
                 #       recreate picking and move again
-                if line.product_id and not line.move_id:
+                if (line.product_id and line.product_id.type != 'service'
```

Comment:
correct, the fix i'm proposing is for the stock_picking_package_preparation_line that comes with the following description:
<<This module adds a new class on Package Preparation to manage details related with stock move or not>>

with this module we can add "details" to the package preparation document, but we do not want to create pickings related to these "details" if the added product is a service.